riscos_toolbox/__init__.py

```diff
@@ -105,9 +73,11 @@ def run(application):
     global _quit
 
     while not _quit:
+        mask = 0 if events.null_polls() else 0b1
```

Comment:
If the poll mask MR goes in first, flags will need to come from the application, and remove the Null mask bit if we need null polls to handle the replies.